console.rb

```diff
@@ -0,0 +1,44 @@
+class Console
+  def console_entry_point
+    puts 'Welcome to my School Library'
+    until list_of_options
+      input = gets.chomp
+      if input == '7'
+        puts 'Thanks for using my School Library App'
+        break
+      end
+      option input
+    end
+  end
+
+  def list_of_options
+    puts
+    puts 'Choose an option by entering a number: '
+    puts '1 List all books'
+    puts '2 List all people'
+    puts '3 Create a person'
+    puts '4 Create a book'
+    puts '5 Create a rental'
+    puts '6 List all rentals for a given person id'
+    puts '7 Exit'
+  end
+
+  def option(input)
+    case input
+    when '1'
+      list_all_books
+    when '2'
+      list_all_persons
+    when '3'
+      create_person
+    when '4'
+      create_book
+    when '5'
+      create_rental
```

Comment:
- [ ] I encountered an issue when I tried creating a rental. Please see the screenshot of the error and try to find a way around it.

Screenshot|
:--------------:|
![a1](https://user-images.githubusercontent.com/39523312/163823643-e5ccc18a-9dc3-4b07-ad6c-4cb7bec9a4f4.png)

- [ ] Secondly, I guess you made a mistake in displaying the personal info of the people.
You did interchange the name to hold the age and the age to hold the name.
Please see screenshot to get the point am trying to make here.

Screenshot|
:-------------:|
![a1](https://user-images.githubusercontent.com/39523312/163824173-6563108c-81b1-4b56-8969-5e81bbf5725e.png)